config/settings/production.py

```diff
@@ -144,6 +144,9 @@
 # ------------------------------------------------------------------------------
 
 
+# SampleSheets
```

Comment:
Very minor nitpick, but usually this is in form of `# Samplesheets app settings` :)